app/i18n/de/index.php

```diff
@@ -53,12 +53,12 @@
 		'starred' => 'Nur Favoriten zeigen',
 		'stats' => 'Statistiken',
 		'subscription' => 'Abonnementverwaltung',
-		'tags' => 'Meine Tags',
+		'tags' => 'Meine Labels',
 		'unread' => 'Nur ungelesene zeigen',
 	),
 	'share' => 'Teilen',
 	'tag' => array(
-		'related' => 'Verwandte Tags',
+		'related' => 'Artikel-Labels',
```

Comment:
This is actually an article tag not a label :)

Reply:
whoops, sorry :)